api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -3304,15 +3315,43 @@ public void purgePatientIdentifierType_shouldThrowErrorWhenTryingToDeleteAPatien
 	}
 	
 	/**
-	 * @verifies fail if not preferred patient has unvoided orders
+	 * @verifies fail if multiple patients has active order of same type
 	 * @see PatientService#mergePatients(org.openmrs.Patient, org.openmrs.Patient)
+	 * @throws APIException
 	 */
 	@Test
-	public void mergePatients_shouldFailIfNotPreferredPatientHasUnvoidedOrders() throws Exception {
+	public void mergePatients_shouldFailIfMultiplePatientsHasActiveOrderOfSameType() throws Exception {
```

Comment:
This method name is a little confusing, can you rephrase it please and actually it would be nice for it to mention drug order in the method name, how about `mergePatients_shouldFailIfBothPatientsHaveActiveDrugOrders`